contrib/src/test/java/com/datatorrent/contrib/cassandra/CassandraOperatorTest.java

```diff
@@ -293,8 +293,8 @@ public void testCassandraInputOperator()
 
     TestInputOperator inputOperator = new TestInputOperator();
     inputOperator.setStore(store);
-    inputOperator.setTablename(TABLE_NAME_INPUT);
     inputOperator.setQuery(query1);
+    inputOperator.setTablename(TABLE_NAME_INPUT);
```

Comment:
Previously, query was dependent on "tableName" variable. As the order of setting variables was tableName followed by query here the test case always passed. To simulate situation where platform could set tablename after query (there is no guarantee in which order variables are set) I did this change.